libs/community/tests/integration_tests/tools/firecrawl/test_tool.py

```diff
@@ -0,0 +1,64 @@
+from typing import Generator
```

Comment:
could you also add some tests for calling the tool with `tool.invoke()`?

Reply:
tests added